src/tests/data/kdf/sp800_56a.vec

```diff
@@ -2411,3 +2414,342 @@ Salt =
60816951200C93F4807CDEC31FC27474CD46E00E7516958D3EE8A74EC06B26A0C6A58A0CEAD9FD5A4690C7557020C8B96EDE876A762173EB8D532A1E67D98448D24EE3E77BE35355B1562FE80123E169060F2844B3F8EFAA8C06F03E2EAEB17679C9785189CEC997233A6365CBFFF17D9259CD194FFE114BA06DB6447B779014BCFBE8F3A073C2D3297842C4438F18424B6BB5BEC36E7162BC19C54CA2F21066FFA7F5E3423CE847189088C160CD6465BA58378BDD8FBCB1DF2B3089BADA64B13AA291A8F5D29B82F6924E7F1F518DECA3802706A44A099150EAB6DEBFDFDA6610A812EDB908E5E15FDCD5E7210C9B9609EF69944174594C904BB1DBB4FB5ECB7A6F8CBBE797DB0C14687BFE979ADC01C56BD91EAB54D69C63FA859514FA4ADBDA6C81FD0E6102E30E14E81711BD4615E0C6D5FFD63938FB6D9A1DE97BEA20FD808CBCDEB0BDC79B6BB057E34F433781E3007F35F8CB68964A31BBD5FC6D9466F7385B483E2A1660FBCECB3ED0832D287019570F501836DBD5FB08C61EC36A36EF28667C6CA79A3A0FD04E2FA73D9253F7D99966A4838E5EB5C7F0D96526304AFE727C62D625020A80FB93DE77EBA3C6D46DEC4E0E68A96B8A3DA16EB11B36A2AEC61B4A0366C0E428A7CF4DEF189C5736A2F59BC8DF1019BC80C996092AB14F4982F71343D968C908B52E1E8166EC53D3AB99636B8025A6D95E236C30C99957C650B4DB6B6F02CF05601BEE86EA77E0ECF960A28178A0FD5E638345A86E2A24A3B409F946D84D344D1BDB0B7E84095DA3CCDA5BEE3D0BCF4AFBB4B9A350EBE16B99728F481BADC6544EDF8D47EAF8131C2795944F96BE16445115F49FA2822DA923F67797268439602AF1AC5350215FEF545B267D9404621009B6CD550EF145FA32DD684AE868774D132854F8CF20FA1E6A0291E87D60868C405FFF9ED24C897EE97B734A3FCAF1E543A3DE1A91A37009549FA2976F2F76A628A815D94EBBF0F143E59B3F294EBE0C990017F306AE3456384D59C33ECC6484BB9C98FDD78AE65B96E5C9C6F9A857AD63E063D4B43142C1A6EC64341687D60CAC94E881FC21EDFCD5CB25496FDB429D82704B15D81119A31E951FC487BC26BB2E39A13B45DA1EAEF53CE5CED46156C5B1EBF13D4305454DD02D5B1AD0E238BB9AF9AAC6A0F38873AD8F1A1E5EEAD079AFEFA7C077C1B5D9FF422FCE4ECB3D0F5B44B8A5DA06B458AB69FEF9DE459000FCA9E60EF29300529F85CC554B34DDAC88F964EFEACD2396FD99547813555FCB977B032B77C609BF33A372FE9CB2F200E1124F3E11277BFFD55ED4FB5334F6128369083E34BEBED737751666CCF8F4D295A7AC0A68E3359FB937EA369E40344A093FF6918488A78D1DB72146C73437BF94733C877E00527AFA8CC6DC4CDB8F59BBDE9A04B78945D22E2B0BBA5D2F2C7F1437122D99F029A444AE1ACA5AD6B928717F76F4B34BF6A6A1D677A8A59629145720B2060400D831D34E50AE2FE7A9A6D3B051864484A0C63D162E7121FCF3945D4AB1037C8CD9C305BA78541233BDCF4899A4992B865E09242F92DDC3DDEC4BA262E2EDB6F14078497C0DA38DC4F5CFF3F24F0E4896F73F3768F5BC04233F125D06180803B0C198A8EC2F22C596721AEB970003F04A52BB6AE69D788AE864799FD33900DFE167F3F540589A526125D4E180A4699CDFE59706F7E96BA4B7D800878184EB6A539A90019CE3D6E259B35451413063E8ECFB24BFA45479726890D9163AF405B6771612E38D569804A43C5B715BD43491EF3B7C873A6381667D1E5735E77B1EB424AF2575C2522468825F2225BAF790C0C5C19259A92A669D8CD435B2BF1A63C7611568EC94FA52829425B1A0010D3AB3D0164F4B4380CA8D9CBA6F14D4B88D34002332F4AA61D7150CD5D973B70258FCE80A6A4D4F64EB940A134D6C0F5E576D9868A3590923BE3B639EAF2E15D4A0EA267F732C26269CA37D3C17699C3BB4BC04EE4758EC7E7B5B379173EB0869D1227DEAA7C31A5B5CDFC190D8847EF6CCD78AB797E6F61ED4CD438D8882055E3455C1965F69123F739CD43D76C14CBEE9C01478493CBE92B8DC54B9038EF8301D3752244368B6E04FD4F35678AFFB29
```

Comment:
Some of these test vectors are really long and I don't see much justification for them considering nothing about the KDF varies on the input lengths

Reply:
Agreed. I removed all vectors with excessive lengths!